u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AndroidKeyStoreAttestation.java

```diff
@@ -49,7 +49,10 @@ private AndroidKeyStoreAttestation(Integer keymasterVersion, byte[] attestationC
   }
 
   /**
-   * Parses the key description extension.  Expected format is:
+   * Parses the key description extension.  Note that this method only parses the description
+   * extension in the leaf cert.  It *does not* validate the certificate (or any chain).
```

Comment:
Do you expect it will at some point? The description, "Move closer", almost implies that you do. If so, this would merit a TODO, no? If not, could your description give a little more guidance about where this is headed?